src/Microsoft.AspNetCore.DataProtection.Extensions/Microsoft.AspNetCore.DataProtection.Extensions.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <Description>Additional APIs for ASP.NET Core data protection.</Description>
-    <TargetFrameworks>net46;netstandard1.3</TargetFrameworks>
```

Comment:
It's the top of the package stack so we would also have to cross-compile Microsoft.AspNetCore.DataProtection and .Abstractions to netstandard1.3 to make that happen.